doc/source/ray-core/doc_code/deser.py

```diff
@@ -0,0 +1,15 @@
+import ray
+import numpy as np
+
```

Comment:
Nit (and not blocking this PR): We are not consistent on whether to add `ray.init()` in code snippets in docs.

Reply:
I think we should avoid, since it's done implicitly now and makes the code longer.